cli/mainrt.rs

```diff
@@ -70,6 +71,22 @@ fn unwrap_or_exit<T>(result: Result<T, AnyError>) -> T {
   }
 }
 
+fn load_env_variables_from_env_file(filename: Option<&String>) {
+  if let Some(env_file_name) = filename {
+    match from_filename(env_file_name) {
+      Ok(_) => (),
+      Err(error) => {
+        match error {
+            dotenvy::Error::LineParse(line, index)=> eprintln!("{} Parsing failed within the specified environment file: {} at index: {} of the value: {}",colors::yellow("Warning"), env_file_name, index, line),
+            dotenvy::Error::Io(_)=> eprintln!("{} The `--env` flag was used, but the environment file specified '{}' was not found.",colors::yellow("Warning"),env_file_name),
+            dotenvy::Error::EnvVar(_)=> eprintln!("{} One or more of the environment variables isn't present or not unicode within the specified environment file: {}",colors::yellow("Warning"),env_file_name),
+            _ => eprintln!("{} Unknown failure occurred with the specified environment file: {}", colors::yellow("Warning"), env_file_name),
+          }
+      }
+    }
+  }
+}
```

Comment:
The code for this already exists elsewhere. Maybe it should be extracted out to a common function and reused here?

Reply:
Hello @dsherret ,
Code has been wrapped within function to reuse.
Checks can be triggered.